worker/include/Utils.hpp

```diff
@@ -1,7 +1,14 @@
 #ifndef MS_UTILS_HPP
 #define MS_UTILS_HPP
 
+#ifndef MS_CLASS
+#define MS_CLASS "Utils"
```

Comment:
We cannot define this macro in a hpp file but just in cpp files so it's local to that cpp file. Otherwise every cpp file importing útiles has its MS_CLASS macro overrided. This breaks everything.  Please move these utils that need logging to the corresponding Utils/Xxxxx.cpp file.

Reply:
BTW, why not using `#pragma once`? It is supported by all compilers we likely care about and from what I read can even help with compilation speed a bit.

Reply:
@nazar-pc  `#pragma once` is non-standard feature and there is no guarantee that the compiler(gcc, clang, microsoft c++) will still support this feature.

Reply:
Theoretically yes, practically it was there for a long time and is not going anywhere. It is even present in files generated by Microsoft's IDEs if what I read is to be believed. So I think the risk of using it is extremely close to 0.

Reply:
What is #pragma?

Reply:
Guys, this does not belong to the PR, sorry. It is a leftover and will be removed.